.travis.yml

```diff
@@ -6,11 +6,12 @@ env:
   global:
   - PACKAGE=sedlex
   matrix:
+  - OCAML_VERSION=4.02 OCAML_SWITCH=4.02.3
   - OCAML_VERSION=4.03
-  - OCAML_VERSION=4.04
-  - OCAML_VERSION=4.05
-  - OCAML_VERSION=4.06
-  - OCAML_VERSION=4.07
+  # - OCAML_VERSION=4.04
+  # - OCAML_VERSION=4.05
+  # - OCAML_VERSION=4.06
+  # - OCAML_VERSION=4.07
```

Comment:
Why did you comment out 4.04 to 4.07?

Reply:
It was temporary, because I expected to need to use Travis to experiment with 4.02/4.03 support differences, and the long development loop (an hour or so) that that imposed was getting in my way. Bit of a hack; but I reverted it when I was done — hence WIP in the pull-request title!